packages/application-components/src/components/modal-pages/tabular-modal-page/README.md

```diff
@@ -19,6 +19,11 @@ Tabular Modal pages are controlled components used to render a page with custom
   onClose={handleClose}
   topBarCurrentPathLabel="Lorem ipsum"
   topBarPreviousPathLabel="Back"
+  formControls={[
```

Comment:
I need my memory refreshed but also don't want to start a long discussion again. In anycase: `components: { ConfirmButton, CancelButton, DeleteButton }` we decided again why?

Reply:
You mean their namings, or the fact that these are the defaults?

Reply:
Their namings. Ah and now I get why we keep it as is. Thanks :)